package.json

```diff
@@ -41,7 +41,7 @@
     "federalist": "npx gulp build",
     "lint": "gulp lintJS lintSass && npm run prettier:scss",
     "prestart": "npx gulp build",
-    "proof": "bundle exec htmlproofer --allow-hash-href=true --enforce-https=false --allow-missing-href=true --ignore-empty-alt=true --ignore-files '/_site/whats-new/updates/2017/'  --ignore-status-codes 403,429 ./_site",
+    "proof": "bundle exec htmlproofer --allow-hash-href=true --enforce-https=false --allow-missing-href=true --ignore-empty-alt=true --ignore-files '/_site/whats-new/updates/2017/'  --ignore-status-codes 403,429,0 ./_site",
```

Comment:
This was just a temporary fix to bypass a broken link in the USWDS repo. Do not include.